lib/errorHandlers/suppressError.ts

```diff
@@ -20,23 +20,17 @@ function createPlatformVersionError(
 ): void {
   let translationKey = 'unspecifiedPlatformVersion';
   let platformVersion = 'unspecified platformVersion';
-  const errorContext = err.data.context;
+  const errorContext = err?.data?.context;
```

Comment:
This line was causing the issue with accessing `.context` on `undefined`, because when we were invoking this function we were passing it `err.data` instead of just `err`, so it was trying to access `err.data.data.context`